ui/explore/src/components/PrometheusMetricsFinder/display/MetricChip.tsx

```diff
@@ -0,0 +1,31 @@
+// Copyright 2024 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Chip, ChipProps } from '@mui/material';
+
+export function MetricChip({ label, ...props }: ChipProps) {
+  if (label === 'gauge') {
+    return <Chip label={label} color="success" {...props} />;
+  }
+  if (label === 'counter') {
+    return <Chip label={label} color="primary" {...props} />;
+  }
+  if (label === 'histogram') {
+    return <Chip label={label} color="warning" {...props} />;
+  }
+  if (label === 'summary') {
+    return <Chip label={label} color="info" {...props} />;
```

Comment:
Looks like color="info" for summary gives the exact same color as color="primary"?
![image](https://github.com/user-attachments/assets/e00fd017-f652-432a-aa19-c28286847f45)
 We could change to a different one for summaries for better differentiation

Reply:
Yup
![image](https://github.com/user-attachments/assets/3395b8a5-10a1-4b51-9c55-7a383bc85370)